### ISSUE TITLE
Getting ReferenceError: HTMLCanvasElement is not defined

### ISSUE THREAD

**Issue by DSoe** [sprmn/react-firebase-file-uploader#5]
Hello, 

First of all, thanks so much for this file uploader package.  
With the version 2.1.1, I am getting the following error when I start my node server.

node_modules/react-firebase-file-uploader/lib/polyfill.js:3
[2] if (!HTMLCanvasElement.prototype.toBlob) {
[2]      ^
[2]
[2] ReferenceError: HTMLCanvasElement is not defined

So, for now, I am using version 2.1.0. 

Thank you,

**Comment by sprmn** [comment]
Woops, totally missed this issue. I'll see if I can fix this asap!

**Comment by sprmn** [comment]
@DSoe Should be fixed in [this branch](https://github.com/sprmn/react-firebase-file-uploader/tree/fix/ssr-polyfilling). Can you check if it works for you? 

FYI, you can test it with
```
yarn add git://github.com/sprmn/react-firebase-file-uploader.git#fix/ssr-polyfilling
```
or 
```
npm i --save git://github.com/sprmn/react-firebase-file-uploader.git#fix/ssr-polyfilling
```

**Comment by DSoe** [comment]
Thanks for the fix @sprmn.

I am getting the following error during the build time using the fix/ssr-polyfilling branch.

```
ERROR in ./src/containers/AdminOrders/ProductForm.js
Module not found: Error: Can't resolve 'react-firebase-file-uploader' in '/Users/dsoe/workspace/React/react-redux-universal-shop/src/containers/AdminOrders'
resolve 'react-firebase-file-uploader' in '/Users/dsoe/workspace/React/react-redux-universal-shop/src/containers/AdminOrders'
```

Thank you,

**Comment by sprmn** [comment]
Sorry, my bad. Forgot to add the build files to that branch. Should be fixed now. Can you retry with a reinstall of the package?

**Comment by DSoe** [comment]
Yep. This time, it works now. @sprmn 

Thank you,

**Comment by sprmn** [comment]
Thanks for testing! Shipped in `2.1.2`!
